apps/common/controllers/add-another-address-loop.js

```diff
@@ -35,16 +12,17 @@ module.exports = class AddAnotherAddressController extends BaseController {
   }
 
   removeItem(req, res) {
-    const items = req.sessionModel.get(`${this.field}Addresses`);
-    req.sessionModel.set(`${this.field}Addresses`, _.omit(items, req.params.id));
-    const step = _.size(items) > 1 ? `/${this.field}-add-another-address` : `/${this.field}-postcode`;
+    const items = req.sessionModel.get(`${this.options.locals.field}Addresses`);
+    req.sessionModel.set(`${this.options.locals.field}Addresses`, _.omit(items, req.params.id));
+    const step = _.size(items) > 1 ?
+      `/${this.options.locals.field}-add-another-address` : `/${this.options.locals.field}-postcode`;
     return res.redirect(`${req.baseUrl}${step}`);
   }
```

Comment:
yeah, I touched on this in my review - removing an item is shared functionality on every page so should be abstracted to the base address controller